pages/docs/community/styleguide/inclusive-language.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Inclusive Language style guide
+description: This style guide outlines the guideline for using inclusive languages in documentation.
+---
+
+# Inclusive Language
+
+While writing documentation we are indirectly reaching out to thousands of people all around the globe. 
+Thus, we must ensure that our diverse audience can connect with our information. 
+
+### 1. Culturally Inclusive Language
+
+Some phrases and words are popularly used only in specific regions of the world, thus our audience from other parts 
+of the world might need help to relate to the same. Hence, we should try to avoid using such phrases in our documentation. 
+Some examples of Region Specific Language are -
+- “It’s not rocket science” is commonly used in the USA, and non-US people might not be able to relate to it properly.
+- Phrases written in regional languages instead of English.
+- “It’s not a piece of cake” is commonly used in the USA, and non-US people might not be able to relate to it properly.
+
+### 2. Gender Neutral Language - 
+
+Some phrases and words are targeting men or men/women specifically. This might result in other gender groups feeling 
+left out while reading our documentation. 
+Some examples of Gender Neutral Language are -
+- Using “they/them” instead of “his/him” or “she/her”.
+- Using “Hello everyone” instead of “Hello guys”.
+- Using “Chairperson” instead of “Chairman” or “Chairwoman”.
+
+### 3. People First Language - 
+
+The placement of the word “People” matters a lot when it comes to writing documentation. 
+Using the word “People” in the beginning makes it more pleasant as the problem is not displayed in the beginning.
+Some examples of Gender Neutral Language are -
+- Using “People with Mental Instability” instead of “Mentally Unstable People”.
+- Using “People with Audio Impairment” instead of “Deaf People”.
```

Comment:
```suggestion
- Using “Person with deafness” instead of “Deaf”.
```

Unfortunately, this example is not inclusive either. 😄 Individuals who are deaf or hard of hearing do not see their condition as a disability but rather as a **cultural identity or a unique way of experiencing the world**. Therefore, it's important to be mindful of the language we use to describe individuals with hearing loss.  They view themselves as members of a linguistic and cultural minority group that uses sign language as their primary means of communication. (Not to mention, the phrase "audio impairment" is not used.)

**Deaf** is the correct and inclusive term, so this should actually be listed as a correct example.